include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -82,6 +82,14 @@ struct explicit_wait_if<true>
     }
 };
 
+template <typename Op, ::std::size_t CallNumber>
+struct unique_kernel_name
+{
+};
+
+template <typename Policy, int idx>
+using new_kernel_name = unique_kernel_name<typename ::std::decay<Policy>::type, idx>;
```

Comment:
Could you, please, add uglification for `new_kernel_name` and `unique_kernel_name`?